api/src/main/java/org/openmrs/module/fgh/mpi/miscellaneous/TokenInfo.java

```diff
@@ -0,0 +1,79 @@
+package org.openmrs.module.fgh.mpi.miscellaneous;
+
+import org.apache.commons.lang.time.StopWatch;
+
+public class TokenInfo {
+	
+	private String access_token;
```

Comment:
By the way this is anti Java, fields are supposed be camel and not snake cased, i.e. accessToken, idToken etc, please fix this too

Reply:
Done